### PR TITLE
Feature/#6 create crud for stop words

### PR DESCRIPTION
**Pull request by ComradeYanis**
Merge "feature/#6 create crud for stop words" with "develop"

### CODE REVIEW

File: app/Http/Controllers/StopWordController.php [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    {
+        StopWord::truncate();
+        return response()->json(['status' => "Items deleted."]);
+
+    }
+
+    public function validator($request)
+    {
+        $rules =  array(
+            'title' => 'required|max:60',
```
Comment (HarisUa) [inline app/Http/Controllers/StopWordController.php]: Title must be unique

File: app/Http/Controllers/StopWordController.php [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    {
+        foreach ($request->title as $value) {
+           
+           $validator = $this->validator($value);
+
+            if(!$validator->fails()) {
+                $new = array(
+                    'title' => $value
+                );
+                $stop_word = StopWord::create($new);
```
Comment (HarisUa) [inline app/Http/Controllers/StopWordController.php]: For what we need create this (`$new`) array? use `$value`

File: app/StopWord.php [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+class StopWord extends Model
+{
+    protected $table = 'stop_words';
+
+    protected $fillable = [
+        'title'
+    ];
+
+    protected $hidden = array('pivot');
```
Comment (HarisUa) [inline app/StopWord.php]: This model has no relationships and dont have `pivot` column.

**Review by HarisUa — CHANGES_REQUESTED** [review]

**Comment by 4rtHurB** [comment]
Can we store by this API array of stop words (not one stop word)?

**Review by HarisUa — APPROVED** [review]

**Pull request merged**